src/hearings/containers/request-hearing/hearing-timing/hearing-timing.component.ts

```diff
@@ -117,13 +116,20 @@ export class HearingTimingComponent extends RequestHearingPageFlow implements On
   }
 
   public setDataItems() {
+    // it is not a requirement to tell the user if the default value of the hearing duration has been updated.
+    if (this.hearingRequestMainModel.hearingDetails.duration && this.hearingRequestMainModel.hearingDetails.duration > 0) {
```

Comment:
Slightly ugly
I would go with
if (this?.hearingRequestMainModel?.hearingDetails?.duration > 0) but I can live with what you wrote